Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLink.cs

```diff
@@ -100,6 +100,7 @@ public class AudioLink : MonoBehaviour
 
         [Header("Experimental (Limits performance)")]
         [Tooltip("Enable Udon audioData array. Required by AudioReactiveLight and AudioReactiveObject. Uses ReadPixels which carries a performance hit. For experimental use when performance is less of a concern")]
+        [Obsolete("Do not reference this field directly, use AudioLink.EnableReadback and AudioLink.DisableReadback instead")]
```

Comment:
if this field is manually accessed in editor or by an udon script then it will go out of sync with the camera state.
I see a couple of ways of solving this
- access modifiers such as internal (maybe even different access modifiers based on UNITY_EDITOR, although that is a little cursed)
- Obsolete attribute, though it seems like weak security
- Guarding the camera disabling logic with !UNITY_EDITOR

Reply:
I also added some buttons to the UI to toggle this state instead

Reply:
lmk what you think @pema99

Reply:
if I do this then we can't reference the field ourselves

Reply:
or at least I don't know how